doc/faq/errors.rst

```diff
@@ -68,3 +68,12 @@ to run the simulation::
 
 
 Try running on more nodes and/or devices.
+
+
+What can I do about ``CUDA_ERROR_FILE_NOT_FOUND: file not found`` errors?
+-------------------------------------------------------------------------
+
+The reason for these errors is unknown, they seem to be more common with higher
+rank counts. A theory is that there is a race condition between caching
+a compiled kernel on disk and the loading of that kernel from that cache.
+The only known "fix" at the moment is to restart the application.
```

Comment:
Do you know which CUDA call issues that error code?

Reply:
Also, it sounds like we should have a tracking issue for this problem, even if it's intermittent.

Reply:
> Do you know which CUDA call issues that error code?

No, this is fiendishly hard to debug as it is completely spurious :-/

Reply:
> Also, it sounds like we should have a tracking issue for this problem, even if it's intermittent.

I looked again and it turns out we have already an issue, although with not much info: https://github.com/illinois-ceesd/mirgecom/issues/679

Reply:
Thanks for digging up the tracking issue!